pages/info/[side].js

```diff
@@ -12,22 +16,56 @@ import PageView from '../../components/PageView';
 
 type RenderProps = WithDataAndLayoutProps<Side_info_QueryResponse>;
 
-const Index = ({ props }: RenderProps) => (
-  <>
-    {props.page ? (
-      <PageView page={props.page} />
-    ) : (
-      <>
-        <h1>Finner ikke siden :( </h1>
-        <h2>404 Errr</h2>
-      </>
-    )}
-  </>
-);
+const StyledHeading = styled('h1')`
+  font-weight: 200;
+  font-size: 4em;
+  margin: 0;
+`;
+
+const StyledPageView = styled('div')`
+  margin-left: 0;
+`;
+
+const Index = ({ props }: RenderProps) => {
+  const { page, pages } = props;
+  const navitems = pages.map(page => ({
+    text: page.title || '',
+    href: '/info/[side]',
+    as: `/info/${page.slug}`,
+    key: page.id
+  }));
+  return (
+    <>
+      <StyledHeading>INFO</StyledHeading>
+      <Navbar items={navitems} />
+      {page ? (
+        <StyledPageView>
+          <PageView page={page} />
+        </StyledPageView>
+      ) : (
+        <>
+          <Flex center full>
+            <FlexItem>
+              <CenterIt text>
+                <h1>Finner ikke siden :( </h1>
+                <h2>404 Errr</h2>
+              </CenterIt>
+            </FlexItem>
+          </Flex>
+        </>
+      )}
+    </>
+  );
+};
 
 export default withDataAndLayout(Index, {
   query: graphql`
     query Side_info_Query($side: String!) {
+      pages(infopage: true) {
```

Comment:
Maybe cleaner to separate this into a separate component (and query-fragment), but this does indeed work.

Reply:
I tried, but couldn't make it work, so I ended with this.